changelog/unreleased/issue-2569

```diff
@@ -0,0 +1,10 @@
+Enhancement: Support excluding files by their size
+
+Previously `backup` command did not support excluding large files by their size, which may be annoying when there are
```

Comment:
I'd like to start the changelog with a short description of the new feature before explaining the solved problem, for example like this:
```
The `backup` command now supports the `--exclude-larger-than` option to exclude files which are
larger than the specified maximum size. This can for example be useful to exclude unimportant
files with a large file size.
```